server/src/main/java/com/ca/lsp/core/cobol/preprocessor/sub/document/impl/SynchronousCopybookResolution.java

```diff
@@ -113,7 +111,8 @@ public void observerCallback(@Nonnull FetchedCopybookEvent event) {
     String content = event.getContent();
 
     broker.unSubscribe(this);
-    ofNullable(content).ifPresent(it -> broker.storeData(new CopybookStorable(name, uri, content)));
+    CopybookModel copybookModel = new CopybookModel(name, uri, content);
+    ofNullable(content).ifPresent(it -> broker.storeData(copybookModel));
     waitForResolving.complete(new CopybookModel(name, uri, content));
```

Comment:
can we safely reuse the instance of ```CopybookModel``` instead to create a new one?

Reply:
Oops, my fault, I was planned to use the same `copybookModel` instance.